frontend/packages/core/src/stepper.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import {
+  Grid,
+  Step as MuiStep,
+  StepConnector as MuiStepConnector,
+  StepLabel as MuiStepLabel,
+  Stepper as MuiStepper,
+} from "@material-ui/core";
+import MuiCheckIcon from "@material-ui/icons/Check";
+import PriorityHighIcon from "@material-ui/icons/PriorityHigh";
+
+const Circle = styled.div((props: { background: string; border: string }) => ({
+  backgroundColor: props.background,
+  border: props.border,
+  boxSizing: "border-box",
+  borderRadius: "50%",
+  height: "24px",
+  width: "24px",
+  top: "24px",
+}));
+
+const DefaultIcon = styled.div((props: { font: string }) => ({
+  height: "100%",
+  width: "100%",
+  color: props.font,
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  fontSize: "14px",
+  fontWeight: 500,
+  lineHeight: "18px",
+}));
+
+const CheckIcon = styled(MuiCheckIcon)((props: { font: string }) => ({
+  fill: props.font,
+  padding: "8px",
+}));
+
+const ClearIcon = CheckIcon.withComponent(PriorityHighIcon);
+
+const StepConnector = styled(MuiStepConnector)((props: { completed?: boolean }) => ({
+  ".MuiStepConnector-line": {
+    height: "5px",
```

Comment:
i think the height on figma is 7px

Reply:
oh it depends on which bar you read the height from. the bar example in the last row specifies 5px

Reply:
i think it is supposed to be 7px, left a comment on figma for clarification though